dashboard/app/models/pd/session.rb

```diff
@@ -54,6 +54,14 @@ def formatted_date_with_start_and_end_times
     "#{formatted_date}, #{start_time}-#{end_time}"
   end
 
+  def american_date_with_start_and_end_times
+    start_date = start.strftime('%b %d %Y').strip
+    start_time = start.strftime('%l:%M%P').strip
+    end_time = self.end.strftime('%l:%M%P').strip
```

Comment:
![facepalm_fozzie](https://user-images.githubusercontent.com/1615761/52653635-324e8900-2ea5-11e9-8185-ff28cd8ffd18.gif)
Not your change: Why'd we use a Ruby keyword as a column name?

Reply:
We can all learn from that :D. I also change function name to _start_date_with_start_and_end_times_us_format_ so other people don't have to read its code to understand what it does.